Index: libAFPDFLib/AFPDFDoc.cpp
===================================================================
--- libAFPDFLib/AFPDFDoc.cpp	(revision 65)
+++ libAFPDFLib/AFPDFDoc.cpp	(working copy)
@@ -646,6 +646,13 @@
 		this->Dispose();
 		gDestroyMutex(&this->hgMutex);
 		CloseHandle(hRenderFinished);
+
+#ifdef _MUPDF
+		if(_mupdf != NULL){
+			delete _mupdf;
+			_mupdf=NULL;
+		}
+#endif
 	}
 	
 	void AFPDFDoc::Dispose()
Index: libAFPDFLib/AFPDFDocInterop.cpp
===================================================================
--- libAFPDFLib/AFPDFDocInterop.cpp	(revision 65)
+++ libAFPDFLib/AFPDFDocInterop.cpp	(working copy)
@@ -10,7 +10,8 @@
 }
 AFPDFDocInterop::~AFPDFDocInterop(){
 	if(_ptr>0)
-		delete _ptr;
+		delete (AFPDFDoc *)_ptr;	//when deleting cast _ptr to right type, so that 
+									//destructor of AFPDFDoc is called!
 	_ptr=0;
 	if(_resultsPtr!=0)
 		free(_resultsPtr);
Index: PDFLibNet/PDFWrapper.h
===================================================================
--- PDFLibNet/PDFWrapper.h	(revision 65)
+++ PDFLibNet/PDFWrapper.h	(working copy)
@@ -657,8 +657,8 @@
 		{
 			//Release unmanaged Resources			
 			if(_pdfDoc!=0){
-				_pdfDoc->Dispose();
-				delete _pdfDoc;
+				//_pdfDoc->Dispose();   //Dispose is called by the destructor!
+				delete _pdfDoc;			//So right here!
 			}
 		}
 
